### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by donavanbecker**
### Changes

- Allow for Hiding Devices based off of `DeviceID` instead of `DeviceType`.
- Adds Support for [SwitchBot Meter](https://www.switch-bot.com/products/switchbot-meter).
- Adds Beta Support for [SwitchBot Curtain](https://www.switch-bot.com/products/switchbot-curtain).

### CODE REVIEW

File: src/platform.ts [hunk 1]
```diff
@@ -87,24 +87,29 @@ export class SwitchBotPlatform implements DynamicPlatformPlugin {
...
       if (this.config.options?.curtain) {
         this.config.options.curtain.set_min;
         this.config.options.curtain.set_max;
       }
-    
 
-      if (this.config.options!.ttl! < 120) {
-        throw new Error('TTL must be above 120 (2 minutes).');
+      if (this.config.options!.refreshRate! < 120) {
+        throw new Error('Refresh Rate must be above 300 (5 minutes).');
```
Comment (SwitchBot-Wonderlabs) [inline src/platform.ts:108]: `throw new Error('TTL must be above 120 (2 minutes).');`
My idea is to set the default value to 300s, but users with fewer devices can adjust it to the minimum 120s.

Comment (donavanbecker) [reply src/platform.ts]: Yeah that's a good idea!

File: src/platform.ts [hunk 2]
```diff
@@ -105,7 +105,8 @@ export class SwitchBotPlatform implements DynamicPlatformPlugin {
       }
 
       if (this.config.options!.refreshRate! < 120) {
-        throw new Error('Refresh Rate must be above 300 (5 minutes).');
+    if (this.config.options!.refreshRate! < 120) {
+        throw new Error('Refresh Rate must be above 120(2 minutes).');
```
Comment (SwitchBot-Wonderlabs) [inline src/platform.ts]: `if (this.config.options!.refreshRate! < 120) {`
This  row is repeated...

Comment (donavanbecker) [reply src/platform.ts]: Sorry, I was just committing using the GitHub app. I will get on my computer and get it fixed.

Comment (SwitchBot-Wonderlabs) [reply src/platform.ts]: It's my fault....

In addition, happy new year to you!!!

Comment (donavanbecker) [reply src/platform.ts]: You also!

Comment (donavanbecker) [reply src/platform.ts]: fixed!

**Pull request merged**